### PR TITLE
chore(): ungulpify

### PR DESCRIPTION
**Pull request by boneskull**
- adds types, prettier, lint-staged, CI, usual suspects

typechecking is disabled; fixes for types should be in another PR (likewise formatting)

### CODE REVIEW

**Comment by boneskull** [comment]
didn't run E2E tests

File: babel.config.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "12"
```
Comment (KazuCocoa) [inline babel.config.json:7]: 14?

Comment (boneskull) [reply babel.config.json:7]: indeed

**Review by KazuCocoa — APPROVED** [review]

**Comment by boneskull** [comment]
closing in lieu of a different one due to tooling malfunction

**Pull request closed without merging**